spec/Standard_Distributions.tex

```diff
@@ -1,446 +0,0 @@
-\sekshun{Standard Distributions}
```

Comment:
I'd prefer this chapter to be removed when the documentation for distributions was added, both from a logical groupings perspective and from a desire to only remove things when we have an improvement to add.  My reasoning behind the latter is so that we don't forget about the section itself should something happen between now and adding the documentation for distributions which prevents it from being written in a timely manner.